untl/README.rst

```diff
@@ -0,0 +1,431 @@
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: String
+:Data Constraint: None
+:Qualifier: http://purl.org/NET/UNTL/vocabularies/agent-qualifiers/
+
+
+Publisher
+---------
+
+:Element: publisher
+:Label: Publisher
+:Sub Elements: location_, name_, info_
+:Definition: The publisher is an entity responsible for making the resource available.
+:Comment: Typically, the name of a publisher should be used to indicate the entity. A publisher may be a business, an organization, a service, a government agency, or, rarely, a person. 
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: String
+:Data Constraint: None
+:Qualifier: None
+
+
+Date
+----
+
+:Element: date
+:Label: Date
+:Sub Elements: None
+:Definition: Dates associated with events in the life cycle of the resource. 
+:Comment: As defined in a profile of ISO 8601 [W3CDTF], follows the YYYY-MM-DD format. 
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: Date/time
+:Data Constraint: None
+:Qualifier: http://purl.org/NET/UNTL/vocabularies/date-qualifiers/
+
+
+Language
+--------
+
+:Element: language
+:Label: Language
+:Sub Elements: None
+:Definition: The language(s) of the intellectual content of the resource.
+:Comment: This will be the language(s) in which a text is written or the spoken language(s) of an audio or video. Visual images normally do not have a language unless there is significant text in a caption or in the image itself. Because of the global nature of the Internet, use of this field is recommended. Preferred usage is to utilize a standard schema for language names as defined by ISO639-2 (three letter language codes), followed optionally by a two-letter country code (taken from the ISO 3166 standard).
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: String
+:Data Constraint: http://purl.org/NET/UNTL/vocabularies/languages/
+:Qualifier: None
+
+
+Description
+-----------
+
+:Element: discription
+:Label: Description
+:Sub Elements: None
+:Definition: Both content and physical descriptions of the resource.
+:Comment: Content description may include but is not limited to: an abstract, table of contents, reference to a graphical representation of content, or a free-text account of the content. Physical description will be used for physical dimensions, extent, pagination, process (tintype, daguerreotype, woodcut, etc.), and related physical details of the original item.  
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: String
+:Data Constraint: None
+:Qualifier: http://purl.org/NET/UNTL/vocabularies/description-qualifiers/
+
+
+Subject-Keyword
+---------------
+
+:Element: subject
+:Label: Subject-Keyword
+:Sub Elements: None
+:Definition: The subject or topic of the resource that succinctly describes the content of the resource.  It is expressed by headings, keywords, phrases, or names; or terms for significantly associated  people, places, and events, etc.
```

Comment:
I was looking at that sentence earlier too. It is a little strange, but I think the semicolon is used because it is distinguishing two uses, and in each there is already a comma series.